src/main/java/core/basesyntax/Aquarium.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public class Aquarium {
+
+    private Animal shark = new Shark();
```

Comment:
that's kind of hardcode, let's make possible put in aquarium anything that can swim